### ISSUE TITLE
Tap and hold to repeat backspace.

### ISSUE THREAD

**Issue by CaidenBlock** [dessalines/thumb-key#48]
I tried using this as my daily keyboard for the last 2 days, this is probably the only thing I consider a real "issue" preventing me from using it further rather than an "enhancement"

After ~300ms (Normal 'long press') of holding the backspace, it should be repeated every ~40ms.

Swiping words isn't my thing.

**Comment by dessalines** [comment]
I'll consider it, but only as long as jetpack's long-press detection doesn't slow down the tap or drag detection.

**Comment by CaidenBlock** [comment]
Sounds great

Mar 20, 2023 10:38:41 Dessalines ***@***.***>:

**Comment by GrimPixel** [comment]
Space and enter also need to behave that way.

**Comment by erik55** [comment]
Yes, especially space!!! Or even better: give us a space which does not transform to other characters if pressed several times. For example the space on the "numpad" could be just always creating spaces, no matter how many times you tap on it. Would that be possible and useful in your eyes?

**Comment by dessalines** [comment]
Could you make those as separate issues? Thanks.

**Comment by dessalines** [comment]
This is mostly circumvented by the spacebar swipe delete.

**Comment by HopefulFrog** [comment]
Thumb-key is great, but the lack of this behavior hurts. The layout I use has backspace in the lower left corner where there's no room for swipe gestures. The option to do repeated delete on hold would be very helpful.
